src/index.js

```diff
@@ -1,16 +1,16 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { createStore } from 'redux';
-import { Provider } from 'react-redux';
-import Counter from './components/Counter';
-import counter from './reducers';
+import React from "react";
+import ReactDOM from "react-dom";
+import { createStore } from "redux";
+import { Provider } from "react-redux";
+import Counter from "./components/Counter";
+import counter from "./reducers/index";
 
 // This line instantiates our central Redux store.
 // The `createStore` function receives the reducer
 // that is responsible for updating the store, along
 // with any initial state that we may want the store
 // to start out with (which is none in this case).
-const store = createStore(counter);
+const store = createStore(counter, { count: 0 });
```

Comment:
Pretty sure you don't need to set initial state here since you're also doing it at the top of the reducer/index.js file. (Like do one or the other)